how-to/applying-a-cowboy-to-qastaging.rst

```diff
@@ -0,0 +1,80 @@
+==============================
+Applying a cowboy to qastaging
+==============================
+
+.. note::
+
+    A cowboy is a hotfix applied to an instance, avoiding a proper deployment
+    by applying code changes directly on an instance.
+
+.. caution::
+
+    A cowboy is a valuable tool to either apply urgent hotfixes or test things
+    quickly e.g. in a staging environment.
+
+    As it avoids a proper deployment, it also comes with a couple of negative
+    side effects:
+
+    - a regular deployment overwrites the cowboy
+
+    - the state of the instance is not clearly defined in the current code
+      and/or deployment configuration
+
+    - and probably some more reasons
+
+    All in all a cowboy should be applied only in rare and extraordinary
+    circumstances.
+
+Before you apply a cowboy, please have a quick conversation with your team
+where the reason for the cowboy is thoroughly discussed.
+
+Applying the cowboy
+===================
+
+#. SSH into bastion ``ssh launchpad-bastion-ps5`` and switch to the
+   ``stb-vbuilder-qastaging`` user by running
```

Comment:
Looking at the rest of the stuff below, I'm guessing this should be `stg-launchpad`? Same in the line below this one

Reply:
Great catch. I copied this part from another how-to.. it feels very repetitive to add this kind of information so often, but I also do not like to leave it out, as logging into the bastion might be something everybody is familiar, especially when only joined recently.